python/lsst/meas/algorithms/utils.py

```diff
@@ -246,7 +246,7 @@ def showPsfCandidates(exposure, psfCellSet, psf=None, frame=None, normalize=True
 
                     try:
                         noSpatialKernel = psf.getKernel()
-                    except:
+                    except Exception:
```

Comment:
Please check if there's an obvious exception(s) raised by `psf.getKernel()` here.

Reply:
It is C++ code, with different versions for different PSF classes. And none of it documents any exceptions that I can find.